crates/recursor/src/tests.rs

```diff
@@ -0,0 +1,201 @@
+use std::{
+    net::{IpAddr, Ipv4Addr},
+    time::Instant,
+};
+
+use hickory_proto::{
+    ProtoError,
+    op::{Message, Query, ResponseCode},
+    rr::RecordType,
+};
+use hickory_resolver::{Name, config::ProtocolConfig};
+use test_support::{MockNetworkHandler, MockProvider, MockRecord, MockResponseSection, subscribe};
+use tokio::runtime::Builder;
+
+use crate::{Recursor, RecursorBuilder};
+
+#[ignore]
```

Comment:
Can you add a comment with the `ignore` here and on L69 (or in the commit message)? 

Alternatively, maybe it makes more sense to land the behaviour updates up-front and then the tests of the new behaviour afterwards?